doc-content/enterprise-only/optimizer/business-optimizer-test-solver-quarkus-proc.adoc

```diff
@@ -1,8 +1,8 @@
 
-[id='business-optimizer-test-application-proc_{CONTEXT}']
-= Test the application
+[id='business-optimizer-test-solver-proc_{CONTEXT}']
+= Test the school timetable solver
 
-A good application includes test coverage. This example tests the {PLANNER} school timetable project on {QUARKUS}. It uses a JUnit test to generate a test dataset and send it to the `TimeTableController` to solve.
+This example tests the {PLANNER} school timetable project on {QUARKUS}. It uses a JUnit test to generate a test data set and send it to the `TimeTableController` to solve.
```

Comment:
```suggestion
This example tests the {PLANNER} school timetable project on the {QUARKUS}. It uses a JUnit test to generate a test data set and send it to the `TimeTableController` to solve.
```

Reply:
Actually, it just {QUARKUS}  - Red Hat build of Quarkus. No the. that goes for all of the Red Hat build ofs... I know, it seems wrong handed..